resources/views/projects/create.blade.php

```diff
@@ -10,124 +10,91 @@
 			</div>
 			<input id="imgInput" name="projectImage" type="file" style="display:none" accept="image/x-png,image/jpeg,image/png" onchange="updateImage(this)">
 		</div>
-		<!-- Project Name -->
-		<div class="field">
-			<label for="projectName">Project Name</label>
-			<input type="text" name="projectName" placeholder="Project Name">
+		<!-- Project name -->
+		<div class="field {{ $errors->has('projectName') ? 'error': '' }}">
+			<label for="projectName">Project name</label>
+			<input type="text" name="projectName" placeholder="e.g. Best Project">
 		</div>
-		<!-- Associated Team -->
-		{{-- <div class="field">
-			<label for="teamName">Associated Team</label>
-			<div class="fields">
-				<div class="seven wide field">
-					<select name="teamName" class="ui search dropdown">
-						<option value="">Existing team</option>
-						@if(isset($teams))
-							@foreach($teams -> all() as $teams)
-								<option value={{$team['id']}}>{{$label['name']}}</option>
-							@endforeach
-						@endif
-					</select>
-				</div>
-				<div class="two wide field">
-					<br>
-					<p id="or">or</p>
-				</div>
-				<div class="seven wide field">
-					<input type="text" name="createTame" placeholder="New Team Name">
-				</div>
-			</div>
-		</div> --}}
-		<!-- Associated Course -->
-		<div class="field">
-			<label for="courses">Associated Course</label>
+		<!-- Associated course -->
+		<div class="field {{ $errors->has('courses') ? 'error': '' }}">
+			<label for="courses">Associated course</label>
 			<select name="courses" class="ui search dropdown">
-				<option value="">Web Development Class</option>
+				<option value="">e.g. Web development class</option>
 				@if(isset($courses))
-					@foreach($courses -> all() as $course)
-						<option value={{$course['id']}}>
-						{{$course['name']}}</option>
+					@foreach($courses->all() as $course)
+						<option value={{ $course->id }}>{{ $course->name }}</option>
 					@endforeach
 				@endif
 			</select>
 		</div>
 		<!-- Labels -->
-		<div class="field">
+		<div class="field {{ $errors->has('label') ? 'error': '' }}">
 			<label for="label">Labels</label>
-			<select name="label[]" multiple="" class="ui fluid dropdown">
-				<option value="">Finance</option>
+			<select name="label" multiple class="ui search dropdown">
+				<option value="">e.g. Finance</option>
 				@if(isset($labels))
 					@foreach($labels -> all() as $label)
-						<option value={{$label['id']}}>{{$label['name']}}</option>
+						<option value={{ $label->id }}>{{ $label->name }}</option>
 					@endforeach
 				@endif
 			</select>
 		</div>
 		<!-- Skillset -->
-		<div class="field">
-			<label for="skillsets">Skillsets</label>
-			<select name="skillsets[]" multiple="" class="ui fluid dropdown">
-				<option value="">Java, HTML</option>
+		<div class="field {{ $errors->has('skillsets') ? 'error': '' }}">
+			<label for="skillsets">Skillset</label>
+			<select name="skillsets" multiple class="ui fluid dropdown">
```

Comment:
This is multiselect. Class should be same as `labels` input `class="ui search dropdown"`